new-lamassu-admin/src/components/inputs/base/Dropdown.js

```diff
@@ -0,0 +1,29 @@
+import FormControl from '@material-ui/core/FormControl'
+import InputLabel from '@material-ui/core/InputLabel'
+import MenuItem from '@material-ui/core/MenuItem'
+import Select from '@material-ui/core/Select'
+import classnames from 'classnames'
+import React from 'react'
+const Dropdown = ({ label, name, options, onChange, value, className }) => {
```

Comment:
Missing a newline between imports and component declaration. Surprised prettier didn't catch this one.